src/app/components/sectioned-form/index.html

```diff
@@ -11,16 +11,30 @@
   </sky-demo-page-summary>
 
   <sky-demo-page-properties sectionHeading="Sectioned form section settings">
+    <sky-demo-page-property
+      propertyName="heading"
+    >
+    Specifies the section header.
+    </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="active"
       isOptional="true"
     >
     Indicates whether the section is active when the form loads.
```

Comment:
In the description for this property, add "This property accepts `boolean` values."